demos/mediapipe/image_classification/README.md

```diff
@@ -0,0 +1,82 @@
+# MediaPipe Demo {#ovms_docs_demo_mediapipe}
+
+This guide shows how to implement graph using the [MediaPipe](../../../docs/mediapipe.md).
+
+Example usage of graph that contains only one model - resnet:
+
+## Prepare the repository
+
+Clone the repository and enter mediapipe image_classification directory
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server/demos/mediapipe/image_classification
+```
+
+## Download ResNet50 model
+
+```bash
+mkdir -p model/1
+wget -P model/1 https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin
+wget -P model/1 https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml
+```
+
+## Run OpenVINO Model Server
+```bash
+docker run -d -v $PWD:/mediapipe -p 9000:9000 openvino/model_server:latest --config_path /mediapipe/config.json --port 9000
+```
+
+## Run the client:
+```bash
+cd model_server/client/python/kserve-api/samples
+
+python grpc_infer_resnet.py --model_name resnetMediapipe --grpc_port 9008 --images_numpy_path .
+./../imgs.npy --input_name in --output_name out --labels_numpy_path ../../lbs.npy
+Image data range: 0.0 : 255.0
+Start processing:
+        Model name: resnetMediapipe
```

Comment:
With those instructions and log_level DEBUG I get:
[2023-05-31 09:03:03.007][123][serving][debug][modelmanager.cpp:1550] Requesting model: resnet; version: 1.
[2023-05-31 09:03:03.007][123][serving][debug][modelinstance.cpp:1019] Model: resnet, version: 1 already loaded
[2023-05-31 09:03:03.008][123][serving][debug][predict_request_validation_utils.cpp:514] [servable name: resnet version: 1] Invalid shape - Expected: (1,3,224,224); Actual: (1,224,3,224); input name: 0
E20230531 09:03:03.008038   123 modelapiovmsadapter.cc:135] src/mediapipe_calculators/modelapiovmsadapter.cc:135 Inference in OVMSAdapter failed: Invalid input shape - Expected: (1,3,224,224); Actual: (1,224,3,224); input name: 0 code: 61
E20230531 09:03:03.008114    63 modelapiovmsinferencecalculator.cc:79] src/mediapipe_calculators/modelapiovmsinferencecalculator.cc:79 Main Close
[2023-05-31 09:03:03.008][63][serving][debug][mediapipegraphexecutor.cpp:221] Mediapipe failed to execute: INTERNAL: CalculatorGraph::Run() failed in Run:
Calculator::Process() for node "ModelAPISideFeedCalculator" failed: ; RET_CHECK failure (src/mediapipe_calculators/modelapiovmsinferencecalculator.cc:150) outputsCount == cc->Outputs().GetTags().size()
E20230531 09:03:03.011585    63 modelapiovmsadapterwrapper.cc:35] Wrapper destr
E20230531 09:03:03.011603    63 modelapiovmsadapter.cc:80] OVMSAdapter destr

Reply:
I think that there is missing layoyt change in subconfig.json